shared/profile/generic/result/container.tsx

```diff
@@ -5,16 +5,8 @@ import Success from '.'
 
 const mapStateToProps = state => ({
   errorText: state.profile.errorCode !== null ? state.profile.errorText || 'Failed to verify proof' : '',
-  proofUsername:
-    state.profile.username + // Auto generated from flowToTs. Please clean me!
-    ((state.profile.platformGenericParams === null || state.profile.platformGenericParams === undefined
-      ? undefined
-      : state.profile.platformGenericParams.suffix) || '@unknown'),
-  // Auto generated from flowToTs. Please clean me!
-  serviceIcon:
-    (state.profile.platformGenericParams === null || state.profile.platformGenericParams === undefined
-      ? undefined
-      : state.profile.platformGenericParams.logoFull) || [],
+  proofUsername: state.profile.username + state.profile.platformGenericParams?.suffic ?? '@unknown',
```

Comment:
`suffix` not `suffic`, right?  wonder why TSC misses it

Reply:
its cause the entire file isn't typed due to not inlined connect. my change above was just fixing all those generated flowToTs things before we had `??`

Reply:
i inlined it and it caught it correctly, fixed